mesonbuild/interpreter.py

```diff
@@ -1974,8 +1979,25 @@ def func_find_program(self, node, args, kwargs):
             progobj = ExternalProgramHolder(extprog)
             if progobj.found():
                 return progobj
-        if required and not progobj.found():
+
+    @permittedKwargs(permitted_kwargs['find_program'])
+    def func_find_program(self, node, args, kwargs):
+        if not args:
+            raise InterpreterException('No program name specified.')
+        required = kwargs.get('required', True)
+        if not isinstance(required, bool):
+            raise InvalidArguments('"required" argument must be a boolean.')
+        progobj = None
+        if self.build.environment.is_cross_build():
+            use_native = kwargs.get('native', False)
+            if not use_native:
+                 progobj = self.program_from_cross_file(args)
```

Comment:
__[flake8]__
* [E111] indentation is not a multiple of four

<!-- Comment by [SideCI](https://sideci.com) -->